tests/tests.py

```diff
@@ -237,60 +240,54 @@ def test_feed_events(self):
 class TestXML(TestCase):
     """Test the validity of the produced XML."""
 
+    data_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                             'data')
+
     @classmethod
     def setUpTestData(cls):
         """Set up the events for use in the tests in this class."""
         cls.event = create_event()
-
-    def test_event_atom_xml(self):
-        """Checks for the correct structure within the Atom XML."""
-        namespace = {'default': 'http://www.w3.org/2005/Atom'}
-        expected_xml_structure = [
-            './default:title',
-            './default:id',
-            './default:updated',
-            './default:author/default:name',
-            './default:content'
-        ]
-        response = self.client.get(reverse('major-event-log:event_atom',
-                                           args=[self.event.id]))
-        atom = ET.fromstring(response.content)
-        for xpath in expected_xml_structure:
-            self.assertIsNotNone(atom.find(xpath, namespace))
+        cls.factory = RequestFactory()
+
+    def test_atom_xml(self):
+        """Validates the Atom items against their schema."""
+        request = self.factory.get('/')
+        response = views.event_atom(request, str(self.event.id))
+        schema_path = os.path.join(self.data_path, 'atom-premis_schema.xsd')
+        with open(schema_path, 'r') as schema_file:
```

Comment:
Absolutely. I was so focused on just getting it to work I forgot my DRY principles.